orc8r/gateway/python/magma/magmad/metrics_collector.py

```diff
@@ -167,7 +174,22 @@ def _add_uptime_metric(self, service_name, family):
         start_time = family.metric[0].gauge.value
         uptime = _get_uptime_metric(service_name, start_time)
         if uptime is not None:
-            self._samples.append(uptime)
+            self._samples_for_service[service_name].append(uptime)
+
+    def _make_sync_done_func(self, service_name, chunk):
+        return lambda future: self._loop.call_soon_threadsafe(
+                               self.sync_done, service_name, chunk,
+                               future)
+
+    def _chunk_samples(self, samples):
+        sample_size_bytes = sys.getsizeof(samples)
+        buckets = math.ceil(
+          sample_size_bytes / self.grpc_max_msg_size_bytes)
```

Comment:
minor nit: should we leave any space for any additional overhead ? can we typically send grpc_max_msg_size_bytes size of msg info without any issues ?